x/docs/md/track/TRY.md

```diff
@@ -14,9 +14,17 @@ In order to be able to submit your solution, you'll need to configure the client
 exercism configure --key=YOUR_EXERCISM_KEY
 ```
 
-When you've written a solution, submit it to the site.
+When you've written a solution, a source file to pass the test suite of a problem, submit it to the site.
 You'll have to configure the command-line client with your exercism API key before you can submit.
 
+Submit the solution with the following command:
+
 ```plain
-exercism submit PATH_TO_FILE
+exercism submit PATH_TO_YOUR_SOURCE_FILE
 ```
+
+If you split your solution into several files or have used additional configuration files, submit them too.
+
+It is fine to submit incomplete solutions. If you are struggling with a problem, submit the solution you currently
```

Comment:
I reviewed the documents that we already have and there is already prior art for submitting unfinished solutions. We can either link to that anchor here or we can repeat the text to keep the wording consistent. 

https://github.com/exercism/exercism.io#are-you-stuck-trying-to-solve-an-exercise